packages/shared/src/components/atoms/LedgerIllustration.svelte

```diff
@@ -4,6 +4,9 @@
 
     export let pill: { localeKey: string; color: string }
```

Comment:
Let's add a default value of undefined here, so no ts errors are shown in VS Code and add the `Color` type from `bloomwalletio/ui`.

```suggestion
    export let pill: { localeKey: string; color: Color } | undefined = undefined
```